Project.toml

```diff
@@ -0,0 +1,19 @@
+name = "IndirectArrays"
+uuid = "9b13fd28-a010-5f03-acff-a1bbcff69959"
+
```

Comment:
We'll need a `version` number here.

Reply:
what version number though?  i left it out because i thought additional PRs might be merged before a new release is tagged, and depending on their content it's hard to know whether the major, minor, or point number would be bumped.